unidist/core/backends/mpi/core/worker/request_store.py

```diff
@@ -178,7 +178,7 @@ def process_wait_request(self, data_id):
             self.put(data_id, communication.MPIRank.ROOT, self.REQ_WAIT)
             logger.debug("Pending wait request {} id".format(data_id._id))
 
-    def process_get_request(self, source_rank, data_id):
+    def process_get_request(self, source_rank, data_id, is_blocked=False):
```

Comment:
```suggestion
    def process_get_request(self, source_rank, data_id, is_blocking_op=False):
```

Reply:
Also, please add a docsting for the parameter like
```
is_blocking_op : bool, default: False
    Whether the get request should be blocking or not.
    If ``True``, the request should be processed immediatly
    # even for a worker since it can get into controller mode.
```